frontend/i18n-scripts/i18n-to-po.js

```diff
@@ -0,0 +1,138 @@
+const fs = require('fs');
```

Comment:
It's hard to review this file since we've lost the diff due to the rename. Can you let us know what's changed?

Reply:
I outlined it in the description: https://github.com/openshift/console/pull/7030#issue-512414140